docs/guide/mocking.md

```diff
@@ -168,6 +168,62 @@ The following principles apply
 * All objects will be deeply cloned
 * All instances of classes and their prototypes will be deeply cloned
 
+### Mocking Pitfalls
+
+Beware that it is not possible to mock calls to methods that are called inside other methods of the same file. For example, in this code:
+
+```ts
+export function foo() {
+  return 'foo'
+}
+
+export function foobar() {
+  return `${foo()}bar`
+}
+```
+
+It is not possible to mock `foo` method from the outside because it is referenced directly. So this code will have no effect on `foo` call inside `foobar` (but it will affect `foo` call in other modules):
```

Comment:
```suggestion
It is not possible to mock the `foo` method from the outside because it is referenced directly. So this code will have no effect on the `foo` call inside `foobar` (but it will affect the `foo` call in other modules):
```